Setup/Patch/Data/AddBio.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Wtc\Film\Setup\Patch\Data;
+
+use Magento\Customer\Model\Customer;
+use Magento\Eav\Setup\EavSetup;
+use Magento\Eav\Setup\EavSetupFactory;
+use Magento\Eav\Model\Config;
+use Magento\Framework\Setup\Patch\DataPatchInterface;
+use Magento\Framework\Setup\ModuleDataSetupInterface;
+use Magento\Customer\Model\ResourceModel\Attribute as
+    AttributeResource;
+
+class AddBio implements DataPatchInterface
+{
+    private ModuleDataSetupInterface $moduleDataSetup;
+
+    private EavSetupFactory $eavSetupFactory;
+
+    private Config $eavConfig;
+
+    /** @var AttributeResource */
+    private AttributeResource $attributeResource;
+
+    public function __construct(
+        EavSetupFactory $eavSetupFactory,
+        Config $eavConfig,
+        ModuleDataSetupInterface $moduleDataSetup,
+        AttributeResource $attributeResource
+    ) {
+        $this->eavSetupFactory = $eavSetupFactory;
+        $this->eavConfig = $eavConfig;
+        $this->moduleDataSetup = $moduleDataSetup;
+        $this->attributeResource = $attributeResource;
+    }
+
+    public function apply()
+    {
+        $eavSetup = $this->eavSetupFactory->create(['setup' => $this->moduleDataSetup]);
+
+        $eavSetup->addAttribute(
+            \Magento\Customer\Model\Customer::ENTITY,
+            'customer_bio',
+            [
+                'type'         => 'varchar',
+                'label'        => 'Bio',
+                'input'        => 'text',
+                'required'     => false,
+                'visible'      => true,
+                'user_defined' => true,
+                'system'       => 0,
+            ]
+        );
+        $customerBioAttribute = $this->eavConfig->getAttribute
+        (Customer::ENTITY, 'customer_bio');
```

Comment:
Line 56-57 should be one line. You can break a method call with long parameter list, but it should be like the following:
```
$myCustomVar = $this->myCustomClass->myMethodWithLotsOfParams(
   $param1,
   $param2,
   $param3,
   ...
   $param4
)
```